generate-task-list.js

```diff
@@ -0,0 +1,55 @@
+const addTaskForm = document.querySelector('.task-form');
+const taskList = document.querySelector('.task-list');
+const addTask = event => {
```

Comment:
And finally there's some important change ready to be made. We need to decouple "adding task" logic from "rendering task list" logic. Create two separate functions with different responsibilities.

1. `addTask`'s responsibilities: handle form submission event, retrieve input values, add a new task item to the `tasks` array, invoke `renderTaskList` function
2. `renderTaskList`'s responsibilities: iterate through the `tasks` array (with `forEach`) and render all items

Most of the code is alreaedy in place, just move all the pieces to proper functions.